readme.md

```diff
@@ -103,72 +119,92 @@ var a = 1;
 console.log(a);
 ```
 
-You're more likely to see `let` declarations inside an `if` or `for` block:
+You're more likely to see `let` declarations inside an `if` or `for` block...
 
 ```js
-//es5
-for (var i = 0; i < 10; i++) {
-  console.log(i)
+// ES5
+var age = 22
+if(age >= 21){
+  var canDrink = true
+} else {
+  var canDrink = false
 }
-console.log("outside loop:", i)
 
-// versus
+console.log(canDrink)
+// => true
+```
 
-//es6
-for (let j = 0; j < 10; j++) {
-  console.log(j)
+```js
+// ES6
+var age = 22
+if(age >= 21){
+  let canDrink = true
+} else {
+  let canDrink = false
 }
-console.log("outside loop:", j)
-//throws an error
-```
 
-The function-scoped of `var` and block-scoped behavior of `let` can have some initially puzzling but important and interesting differences in certain situations. We will delay the `console.log(i)` operation, or evaluating `i`, with `setTimeout` to highlight an important distinction in how **scope** with `let` and `var` operates differently.
+console.log(canDrink)
+// => Uncaught ReferenceError: canDrink is not defined
+```
 
-In the following scenario, since `var` is ***not block-scoped***, it "leaks" outside of the for-loop. This is not the case with `let`, which is **block-scoped**.
+<details>
+  <summary><strong>After the block scope exercises, open this for notes on how <code>let</code> can be useful when writing <code>for</code> loops...</strong></summary>
 
-```js
-//es5
-for (var i = 0; i < 10; i++) {
-  setTimeout(function(){
-    console.log(i)
-  }, 200)
-}
+  The function-scoped behavior of `var` and block-scoped behavior of `let` can have some puzzling but important differences in certain situations. We will delay the `console.log(i)` operation, or evaluating `i`, with `setTimeout` to highlight an important distinction in how **scope** with `let` and `var` operates differently.
 
-// versus
+  In the following scenario, since `var` is ***not block-scoped***, it "leaks" outside of the for-loop. This is not the case with `let`, which is **block-scoped**.
 
-//es6
-for (let j = 0; j < 10; j++) {
-  setTimeout(function(){
-    console.log(j)
-  }, 200)
-}
-```
+  ```js
+  // ES5
+  for (var i = 0; i < 10; i++) {
+    setTimeout(function(){
+      console.log(i)
+    }, 200)
+  }
+  ```
+
+  ```js
+  // ES6
+  for (let j = 0; j < 10; j++) {
+    setTimeout(function(){
+      console.log(j)
+    }, 200)
+  }
+  ```
 
+</details>
 
```

Comment:
I see why you moved this to a details, the IIFE is a clearer or more direct example